src/components/CreateResolution/CompleteResolutionSummary.vue

```diff
@@ -0,0 +1,180 @@
+    >
+        <v-container v-if="!getCreateResolutionStep.validationDetail.valid" class="pl-0 pb-5">
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text ml-1 mr-2">This step is unfinished.</span>
+          <router-link
+            :to="{ path: `/${RouteNames.CREATE_RESOLUTION}` }"
+          >Return to this step to finish it</router-link>
+        </v-container>
+      <v-row class="ml-1 mt-3" no-gutters>
+        <v-col md="3">
+          <strong>Resolution Date</strong>
+        </v-col>
+        <v-col md="9" class="ml-n1">
+          {{resolutionDate}}
+        </v-col>
+      </v-row>
+      <v-row class="ml-1 mt-5" no-gutters>
+        <v-col md="3">
+          <strong>Resolution Text</strong>
+        </v-col>
+        <v-col md="9" class="ml-n1">
+          <v-lazy>
+            <v-textarea
+              id="resolution-text"
+              rows="1"
+              auto-grow
+              readonly
+              filled
+              background-color="white"
+              v-model="resolutionText">
+            </v-textarea>
+          </v-lazy>
+        </v-col>
+      </v-row>
+      <v-row class="ml-1 mt-3" no-gutters>
+        <v-col md="3">
+          <strong>Signing Party</strong>
+        </v-col>
+        <v-col md="9" class="ml-n1">
+          {{signingParty}}
+        </v-col>
+      </v-row>
+      <v-row class="ml-1 mt-5" no-gutters>
+        <v-col md="3">
+          <strong>Date Signed</strong>
+        </v-col>
+        <v-col md="9" class="ml-n1">
+          {{signingDate}}
+        </v-col>
+      </v-row>
+      <v-row v-if="getCreateResolutionStep.resolutionConfirmed" class="ml-1 mt-5" no-gutters>
+        <v-col md="auto" class="mr-2">
+          <v-icon class="upload-success-chk" color="successCheckmark">mdi-check</v-icon>
+        </v-col>
+        <v-col md="11" id="file-name-col">
+          <span>{{ getCreateResolutionResource.confirmSection.reviewSummaryText }}</span>
+        </v-col>
+      </v-row>
+    </section>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { DateMixin } from '@/mixins'
+
+// Enums
+import { RouteNames } from '@/enums'
+import { CreateResolutionIF, CreateResolutionResourceIF } from '@/interfaces'
+
+@Component({})
+export default class CompleteResolutionSummary extends Mixins(DateMixin) {
+  @Getter getCreateResolutionStep!: CreateResolutionIF
+  @Getter getCreateResolutionResource!: CreateResolutionResourceIF
+  @Getter isBaseCompany!: boolean
+  @Getter isTypeCoop!: boolean
+
+  // Enum for template
+  readonly RouteNames = RouteNames
+
+  get resolutionDate (): string {
+    const result = this.getCreateResolutionStep.resolutionDate
+      ? this.dateStrToPacificDate(this.getCreateResolutionStep.resolutionDate, true)
+      : '(Not Entered)'
+    return result
```

Comment:
If you prefer, you can simplify these getters return syntax, 
ie `return this.getCreateResolutionStep.resolutionDate ? someValue : someOtherValue`. 
Not major at all

Reply:
(Try to keep it readable for the reviewers and future devs 😁 )

Reply:
(What I mean is, sometimes an if-then-else is clearer than a ternary.)

Reply:
So I end up using `if-then` for most of the getters in this component and did some earlier variable assignments in an attempt to increase readability.  Let me know if this works for you guys.

Reply:
haha well, personally i don't mind the ternaries. I prefer them over `if-else` when it is simple just an `if-else` scenario.

Reply:
but i'm good with either!

Reply:
Looks more readable now for sure.

Yes, use ternaries for simple scenarios!